homeassistant/components/google_generative_ai_conversation/manifest.json

```diff
@@ -8,5 +8,5 @@
   "documentation": "https://www.home-assistant.io/integrations/google_generative_ai_conversation",
   "integration_type": "service",
   "iot_class": "cloud_polling",
-  "requirements": ["google-genai==1.1.0"]
+  "requirements": ["google-genai==1.5.0"]
```

Comment:
There is already [1.7.0](https://github.com/googleapis/python-genai/releases/tag/v1.7.0)
Can update, rebase, and make sure tests still pass?

Reply:
Upgraded